codegen/src/main/kotlin/org/web3j/openapi/codegen/coregen/subgenerators/CoreApiGenerator.kt

```diff
@@ -90,8 +79,17 @@ internal class CoreApiGenerator(
             }
     }
 
-    private fun contractResources(): List<ContractResource> {
-        val resources = mutableListOf<ContractResource>()
+    private fun eventsImports(): List<Import> {
+        return contractDetails.abiDefinitions
+            .filter { it.type == "event" }
+            .map {
+                Import("import $packageName.core.${contractDetails.lowerCaseContractName}.events.${it.sanitizedName()!!.capitalize()}EventResource")
```

Comment:
The call `it.sanitizedName()!!` can throw an exception, why the function `AbiDefinition.sanitizedName()` can return null? Does `AbiDefinition.name` can ever be null?

Reply:
`AbiDefinition.name` cannot be `null`. I just fixed it. Check now ?